@scottnath Doesn't work with the latest changes to pattern library utilities.

```
➜  generator-pattern-library-test git:(scottnath-minor-fixes) ✗ gulp build
module.js:338
    throw err;
    ^

Error: Cannot find module 'minimist'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/Hyunho/Sites/PatternLibrary/generator-pattern-library-test/node_modules/pattern-library-utilities/lib/gulp-tasks/clone.js:12:16)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
```

--- a/generators/app/templates/_package.json
+++ b/generators/app/templates/_package.json
@@ -12,7 +12,7 @@
     "gulp-sass": "^2.0.4",
     "gulp-task-listing": "^1.0.1",
     "gulp-watch": "^4.3.5",
-    "pattern-library-utilities": "^0.1.2",
+    "pattern-library-utilities": "^0.1.4",
     "run-sequence": "^1.1.2"
   },
   "dependencies": {

